News Theme Development.md

```diff
@@ -133,95 +133,86 @@ Merge into `canary` branch to publish to canary tag. Please reach out to arc blo
+1. Ensure canary is signed off with all RC tickets merged in
+2. `git pull origin canary`
+3. `git push origin canary:rc`
 
-1. Pull the latest `beta` branch:
+Any environment with the `BLOCK_DIST_TAG=rc` will get the updated blocks on next deploy
 
-```sh
-git checkout beta
-git fetch -a
-```
-2. Make a pr against the `beta` branch. After merging, this will push a hashed version to the `beta` tag. You can view the result of the github action work with `npm view [package name]@beta` (eg, `npm view @wpmedia/date-block@beta`).
 
-3. Deploy a bundle with the `BLOCK_DIST_TAG` environment variable set as `beta` in your environment file(s).
-4. After either design QA or product QA approval of that deployed bundle, checkout `stable` and pull down the latest from that branch. Then run `git merge vX.X.X-beta.X` (where `X.X.X-beta.X` is the beta release we're releasing to production) to get the changes from `beta` into `stable`.
-5. Then, in `stable`, you can publish a production release with the following command:
+## Creating a Beta Release
 
-```sh
-npx lerna publish --conventional-commits --conventional-graduate
-```
+Beta Release is the contents of the RC branch once signed off - RC -> Beta
 
-Make sure all blocks have been graduated or promoted.
+1. Enusre RC is ready - All PR's/hotfixes made against RC are merged in
+2. `git pull origin rc`
+3. `git push origin rc:beta`
 
-7. After publishing from the `stable` branch, merge `stable` into `beta` and `beta` into `canary` so that the changes related to the publish we just did end up in both of those branches.
+Any environment with the `BLOCK_DIST_TAG=beta` will get the updated blocks on next deploy
 
-Merging `beta`
 
-```sh
-# Ensure we're on the stable branch
-git checkout stable
-# Ensure we have the latest from stable
-git pull origin stable
-# Checkout beta
-git checkout beta
-# Ensure we have the latest from beta
-git pull origin beta
-# Rebase beta onto stable
-git rebase stable
-# Push the updated beta branch (--force is required or else it will fail)
-git push --force origin beta
-```
+## Creating a stable Release
 
-Merging `canary`
+Stable Release is the contents of the beta branch once signed off - beta -> stable
 
-```sh
-# Ensure we're on the beta branch
-git checkout beta
-# Ensure we have the latest from beta
-git pull origin beta
-# Checkout canary
-git checkout canary
-# Ensure we have the latest from canary
-git pull origin canary
-# Rebase canary onto beta
-git rebase beta
-# Push the updated canary branch (--force is required or else it will fail)
-git push --force origin canary
-```
+1. Enusre beta is ready - All PR's/hotfixes made against beta are merged in
+2. `git pull origin beta`
+3. `git push origin beta:stable`
+4. To release blocks
+    * Clean your local folder and install dependencies - `npx lerna clean -y && rm -rf node_modules && npm i && npx lerna clean -y`
+    * Publish the blocks - `npx lerna publish --conventional-commits --conventional-graduate`
```

Comment:
if all of the blocks are not published, we've also had to go through force-publishing to ensure they are. conventional-commits afaik give us changelogs but we don't use this. the force publish gives us also more flexibility for minor, major and patch releases https://github.com/WPMedia/fusion-news-theme-blocks/wiki/Stable-Publish:-Package-Not-Found-Of-Latest-Version

Reply:
I might recommend the force publish @jgrosspietsch , thoughts?